client/blocks/login/index.jsx

```diff
@@ -433,21 +448,32 @@ class Login extends Component {
 				);
 			}
 		} else if ( isWooCoreProfilerFlow ) {
-			const subtitle = currentQuery.lostpassword_flow
-				? translate(
+			const isLostPasswordFlow = currentQuery.lostpassword_flow;
+			const isTwoFactorAuthnFlow = this.props.twoFactorEnabled;
```

Comment:
Super nitpick: in other places, we have `twoFactorAuthType`, `twoFactorEnabled`, `getTwoFactorAuthNonce`, etc.

I think `isTwoFactorAuthFlow` is more apt 😄

Reply:
hahaha I thought about this back and forth for a moment, but yea for consistency that makes sense